spec/halogen/links_spec.rb

```diff
@@ -68,6 +68,42 @@
   end
 
   describe Halogen::Links::InstanceMethods do
+    describe "options[:include_links]" do
+      let :klass do
+        Class.new do
+          include Halogen
+        end
+      end
+      
+      it "includes links when true" do
+        link = klass.link(:self, :templated, foo: 'foo', attrs: { bar: 'bar' }) { 'path' }
+        repr = klass.new('include_links'  => true)        
+
+        expect(repr.options).to eq({include_links: true})
```

Comment:
{include_links: true} => { include_links: true } since the rest of the code does that too (similarly for the rest)